packages/integrations/cloudflare/functions/[[path]].js

```diff
@@ -0,0 +1,105 @@
+globalThis.process = {
```

Comment:
sorry I did miss this one, can you also delete the functions folder, anything that got build should not be added to the repo

Reply:
Good catch!